src/Symfony/Component/Image/Tests/Issues/Issue59Test.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace Symfony\Component\Image\Tests\Issues;
+
+use Symfony\Component\Image\Fixtures\Loader as FixturesLoader;
+use Symfony\Component\Image\Gd\Loader;
+use Symfony\Component\Image\Exception\RuntimeException;
+use Symfony\Component\Image\Tests\TestCase;
+
+class Issue59Test extends TestCase
```

Comment:
Tests should be renamed, as referencing Imagine issue numbers does not make sense